build/Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+node {
+    stage('Checkout') {
+        checkout scm
+    }
+
+    stage('Setup') {
+        sh """
+        docker exec mattermost-mysql-57 mysql --user=root --password=passwd -e 'create database `'$BUILD_TAG'`;'
+        """
+        echo 'Created database for mysql'
+        sh """
+        # Download and configure Mattermost
+        wget https://releases.mattermost.com/mattermost-platform/master/mattermost-enterprise-linux-amd64.tar.gz
+        tar -zxf mattermost-enterprise-linux-amd64.tar.gz
+
+        cd mattermost
+        sed -i'' 's|mmuser:mostest@tcp(dockerhost:3306)/mattermost_test?charset=utf8mb4,utf8|root:passwd@tcp(localhost:3306)/'${BUILD_TAG}'?charset=utf8mb4,utf8|g' config/config.json;
```

Comment:
I suspect this is from other such files, but I wonder if we should adopt a less brittle approach and just export the desired variables to the environment vs. messing with the config.json syntax:
```
export MM_SQLSETTINGS_DATASOURCE="..."
export MM_TEAMSETTINGS_ENABLEOPENSERVER="true"
...
```

Reply:
agree

Reply:
done and thanks so much for this valuable info!